install/02_machineautoscaler.crd.yaml

```diff
@@ -5,6 +5,8 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.9-0.20200331153640-3c5446d407dd
```

Comment:
Can you remove these two lines please, I realise they're generated by they are unnecessary and will actually break the yaml by defining `annotations` twice